k8s/ithc/common/bar/identity.yaml

```diff
@@ -5,8 +5,8 @@ metadata:
   namespace: bar
 spec:
   type: 0
-  resourceID: "/subscriptions/62864d44-5da9-4ae9-89e7-0cf33942fa09/resourceGroups/managed-identities-demo-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/bar-ithc-mi"
-  clientID: "520446d3-fd1a-40d7-8fae-c8691ec68d81"
+  resourceID: "/subscriptions/7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c/resourceGroups/managed-identities-demo-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/bar-ithc-mi"
```

Comment:
The resourceGroup reference is incorrect. it should be:- /subscriptions/7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c/resourceGroups/managed-identities-ithc-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/bar-ithc-mi